runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpStateSingle.java

```diff
@@ -163,6 +253,17 @@ else if (durationHours > 0)
 		return String.format("%02d:%02d", durationMinutes, durationSeconds);
 	}
 
+	private static Map<String, Integer> loadNpcHealth()
+	{
+		Gson gson = new Gson();
+		Type type = new TypeToken<Map<String, Integer>>()
```

Comment:
please don’t make new Gson instances for every function call

Is there one that can be dependency injected?

This type should also be done once instead if each function call.

Also IO really does not belong in this class